packages/widget-core/src/types.ts

```diff
@@ -113,3 +115,19 @@ export type EventHandlerPayload<T extends keyof EventHandlers, E = Required<Even
 ) => void
 	? Parameters<E>['0']
 	: never
+
+export type Token = {
+	accessToken: string
+	entityId: number
+	expiresIn: number
+	tokenType: 'Bearer'
+	creationDate: number
+	licenseId: number
+}
```

Comment:
Maybe we could describe this type in README?

Reply:
Idk if it's really necessary. I relied on livechat docs, so there is a link to that with more information.
[here](https://github.com/livechat/chat-widget-adapters/pull/72/files#diff-0e14e2a514242459eb4b9d8930b074d1de0b6a665c63a145f58affc2bb82ee1eR199)